tests/import_test.go

```diff
@@ -137,6 +141,83 @@ var _ = Describe("[rfe_id:1115][crit:high][vendor:cnv-qe@redhat.com][level:compo
 	})
 })
 
+var _ = Describe("[Istio] Namespace sidecar injection", func() {
+	var (
+		f = framework.NewFramework(namespacePrefix)
+
+		// FIXME: Istio sidecar injection prevents access to external resources, so we cannot use internal urls (http://cdi-file-host) for the test
+		tinyCoreIsoExternalURL = "http://tinycorelinux.net/12.x/x86/release/TinyCore-current.iso"
+	)
+
+	BeforeEach(func() {
+		By("Enable istio sidecar injection in namespace")
+		labelPatch := `[{"op":"add","path":"/metadata/labels/istio-injection","value":"enabled" }]`
+		_, err := f.K8sClient.CoreV1().Namespaces().Patch(context.TODO(), f.Namespace.Name, types.JSONPatchType, []byte(labelPatch), metav1.PatchOptions{})
+		Expect(err).ShouldNot(HaveOccurred())
+
+		By("Create istio sidecar")
+		sidecarRes := schema.GroupVersionResource{Group: "networking.istio.io", Version: "v1beta1", Resource: "sidecars"}
+		registryOnlySidecar := generateRegistryOnlySidecar()
+		_, err = f.DynamicClient.Resource(sidecarRes).Namespace(f.Namespace.Name).Create(context.TODO(), registryOnlySidecar, metav1.CreateOptions{})
+		Expect(err).ShouldNot(HaveOccurred())
+	})
+
+	It("[test_id:6498] Should fail to import with namespace sidecar injection enabled, and sidecar.istio.io/inject set to true", func() {
+		value := os.Getenv("KUBEVIRT_DEPLOY_ISTIO")
+		if value != "true" {
+			Skip("No Istio enabled, skipping.")
+		}
+
+		dataVolume := utils.NewDataVolumeWithHTTPImport("istio-sidecar-injection-test", "100Mi", tinyCoreIsoExternalURL)
+		By(fmt.Sprintf("Create new datavolume %s", dataVolume.Name))
+		dataVolume.Annotations[controller.AnnImmediateBinding] = "true"
```

Comment:
We have a function called
```
ForceBindIfWaitForFirstConsumer
```
Or
```
ForceBindPvcIfDvIsWaitForFirstConsumer
```
If you are using a DV.

Reply:
In case I use the above, we'll have the consumer pod which will also fail due to the Istio sidecar (already tested). 
I've set the Immediate Binding annotation to true, so afaiu we have no need at all for the consumer pod here.

Reply:
Okay, can you put in a comment indicating why doing the annotation is superior to using the function (the consumer pod is messed up) so that a future self/reviewer can understand this.

Reply:
done